frappe/email/receive.py

```diff
@@ -440,11 +441,19 @@ def set_from(self):
 		self.from_real_name = parse_addr(_from_email)[0] if "@" in _from_email else _from_email
 
 	@staticmethod
-	def decode_email(email):
+	def decode_email(email: bytes | str | None) -> str | None:
 		if not email:
 			return
+		email = frappe.as_unicode(email).replace('"', " ").replace("'", " ")
+		try:
+			parts = decode_header(email)
+		except HeaderParseError:
+			# Fallback: grab just the email addresses
+			emails = re.findall(r"(<.*?>)", email)
+			return ", ".join(emails)
```

Comment:
Should it throw when there is not email? (possibly when the `<` are encoded too?)

Reply:
You mean when it can't decode anything via `decode_header()` or the regex?

Should be fine to just let it be as is

Reply:
Yeah, hypothetically where the regex cannot find `<email@example.com>` because the `<`/`>` are encoded for some reason, but I've never seen it happen

Reply:
Yep then fine as it is, no need of throwing.